docs/topologies/star-shaped.md

```diff
@@ -0,0 +1,51 @@
+# Star-Shaped Head Network
+
+This document details the behaviour of so-called _Star-shaped Hydra Network_.
+
+## Summary
+
+A _Star-shaped Hydra Network_ or more precisely a Star-shaped Heads Network is comprised of:
+
+* A central _Server_ node providing "Head-as-a-service", with low expected downtime, probably operated by some company or organisation with enough resources to host this service,
+* _Client_ nodes, either DApp instances, or mobile/personal wallets, which might not be always online and possibly can come and go.
+
+![Star-shaped Heads Network](./star-shaped-general.jpg)
+
+Client nodes want to be able to interact with each other efficiently, at a low cost, using L2 solution, but without bearing the operational burden of operating a full blown Hydra node. There might be a lot of them, say in the 100s or even 1000s but they are not always all live and up at the same time.
```

Comment:
Not operating a `hydra-node` would go more in the direction of "Managed Hydra Head". Which is likely an orthogonal design and not really relevant for the (star-shaped) networking of Heads, which this document describes.

Reply:
I will rephrase it but that's not exactly what I meant: The clients would be hydra-nodes, but lighter ones, eg. without much customisation and possibly embedded into something else, eg. a wallet.